--- qwt-6.1.3/qwtfunctions.pri.orig	2016-06-13 09:14:23.025600974 +0200
+++ qwt-6.1.3/qwtfunctions.pri	2019-01-06 10:10:07.662538429 +0100
@@ -12,7 +12,11 @@
 defineReplace(qwtLibraryTarget) {
 
     unset(LIBRARY_NAME)
+    greaterThan(QT_MAJOR_VERSION, 4) {
+    LIBRARY_NAME = $${1}$${QT_MAJOR_VERSION}
+    } else {
     LIBRARY_NAME = $$1
+    }
 
     mac:contains(QWT_CONFIG, QwtFramework) {
 
@@ -36,7 +40,11 @@
 defineTest(qwtAddLibrary) {
 
     LIB_PATH = $$1
+    greaterThan(QT_MAJOR_VERSION, 4) {
+    LIB_NAME = $${2}$${QT_MAJOR_VERSION}
+    } else {
     LIB_NAME = $$2
+    }
 
     mac:contains(QWT_CONFIG, QwtFramework) {
 
